events/equeue/equeue.c

```diff
@@ -37,7 +37,7 @@ static inline int equeue_clampdiff(unsigned a, unsigned b) {
 // Increment the unique id in an event, hiding the event from cancel
 static inline void equeue_incid(equeue_t *q, struct equeue_event *e) {
     e->id += 1;
-    if (!(e->id << q->npw2)) {
+    if (0 == (e->id << q->npw2)) {
```

Comment:
nit: Could you flip the order of this comparison? It's easier to read with constants on the right, and all of our compilers have warnings against asignments in if statements anyways.